src/main/java/uk/gov/pay/api/model/ledger/RefundTransactionFromLedger.java

```diff
@@ -0,0 +1,56 @@
+package uk.gov.pay.api.model.ledger;
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import com.fasterxml.jackson.databind.PropertyNamingStrategy;
+import com.fasterxml.jackson.databind.annotation.JsonNaming;
+
+@JsonNaming(PropertyNamingStrategy.SnakeCaseStrategy.class)
+@JsonIgnoreProperties(ignoreUnknown = true)
+public class RefundTransactionFromLedger {
+
+    String gatewayAccountId;
+    Long amount;
+    String description;
+    String reference;
+    String createdDate;
+    String refundedBy;
+    String transactionType;
+    String transactionId;
+    TransactionState state;
```

Comment:
I'd include only fields that will be actually used. The rest is just a noise.

Reply:
We are going to need most of these fields (but not for payment refunds). Removed fields that are not needed for any endpoint